pandas/core/categorical.py

```diff
@@ -199,18 +199,21 @@ class Categorical(PandasObject):
     >>> from pandas import Categorical
```

Comment:
while you are at it, can you remove this line and update the others to use `pd.`

Reply:
Ok